src/components/calculator/PrevalenceControls.tsx

```diff
@@ -207,7 +215,16 @@ export const PrevalenceControls: React.FunctionComponent<{
   const topLocationOptions = Object.keys(locationGroups).flatMap(
     (groupName) => {
       return locationGroups[groupName].map((locKey) => {
-        return { label: Locations[locKey].label, value: locKey }
+        const country_label =
+          Locations[locKey].iso3 &&
+          countries.getName(Locations[locKey].iso3!, i18n.language, {
+            select: 'official',
+          })
+            ? countries.getName(Locations[locKey].iso3!, i18n.language, {
+                select: 'official',
+              })
+            : Locations[locKey].label
+        return { label: country_label, value: locKey }
```

Comment:
![image](https://user-images.githubusercontent.com/32290/105640102-00842200-5e31-11eb-82b1-0a14a191db17.png)

Because we list states first and countries second, I don't know if it's immediately clear to a user that they can type country names in their language. I'm not sure what an easy way around this is. Maybe they just assume since everything else is translated, this will be translated too. But I think seeing "Alabama" as the first option might be a bit confusing.

The only slight improvement I can think of is to show more options in the typeahead by default, so they can scroll down and get a sense of the available options, and they'll soon realize they are translated.

Idea #2: We could also consider changing the placeholder text to have the name of a country where the currently selected language is commonly spoken. That way, they will be seeing it with an example value. Saying "select a country or US state" doesn't actually help that much, since the label above says the exact same thing. Placeholders are usually used as an example of what you could fill in.

Thoughts?

Reply:
Yes, I also saw that to be the problem, though not a huge one.

One quick fix would be to move the US to be the first in the list or to move the US states after the countries. The country names  will be translated, so it will be evident that one needs to use localized names here. I'll try to add a patch that does that.

Idea no.2 is difficult for multiple reasons. For one, locales are about _languages_ and identified by ISO 639-1 two-letter codes. Locations are _countries_, identified by ISO 3166 codes. There's an overlap between these, but there's no 1:1 mapping and no reliable heuristic to convert between them (eg. should "Spanish" be converted to Spain or the country with the most Spanish speakers, or the most populous country that has Spanish as an official language...) Also, it's too easy to mistake the placeholder for the current selection and not understand why the calculator is telling you to select a location, even though one is already selected.

Prefilling using geolocation (#177) would probably be the best solution here.

Reply:
Added a patch that moves the US states to the end:

![image](https://user-images.githubusercontent.com/1048356/105641803-3a334800-5e86-11eb-846a-524a51cd69d4.png)

It's a tradeoff, though: with this, we would be making it more difficult for US users, especially on mobile devices, to select their state as they need to type it in (or scroll endlessly to the bottom).

Reply:
Hmm, looking at it even more I realized an even bigger problem: the options were sorted according to their English names rather than according to their name in the current locale. This could be extremely confusing (case in point: "États-Unis d'Amérique" appeared near Uruguay instead of near Estonia). I updated the code to sort the entries according to the locale. This solves the non-visible translation problem, too, but my previous comment about making it more difficult for US users still applies: it might even be more difficult at this point as states are intermingled between countries, making them harder to find.

Reply:
Hmmm. Dang. Yeah. I hear you on those problems. I'm not clear what the best solution is here. I do think our user base is US-centric, so maybe we just put it back to how it was (with your countries alphabetically sorted by locale-name at the bottom). I'm open to other solutions, but now that I'm seeing the other options, it seems like that's the least-bad option. What're your thoughts?

Reply:
How about this: keep the original order with US states on the top when English language is selected and reorder alphabetically for other languages? We can assume that most non-English speakers come from outside the US, so US states are less relevant for them, so it's not a problem if they're a bit more difficult to find. The only possible caveat I see is with Spanish as that's also widely spoken in the US. We can make an exception for that as well and keep the US states on the top, too, but I'd vote against that: the potential Spanish user base is huge and probably bigger outside of the US.

I pushed a patch that makes this change, you can check it out in the Netlify preview.